articles/azure-monitor/app/correlation.md

```diff
@@ -56,24 +56,35 @@ When the call `GET /api/stock/value` is made to an external service, you want to
 
 ## Correlation headers
 
-We're working on an RFC proposal for the [correlation HTTP protocol](https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/HttpCorrelationProtocol.md). This proposal defines two headers:
+We're transitioning to [W3C Trace-Context](https://w3c.github.io/trace-context/) which defines:
+
+- `traceparent`: Carries the globally unique operation ID and unique identifier of the call.
+- `tracestate`: Carries tracing system-specific context.
+
+Latest versions of Application Insights SDKs support Teace-Context protocol, but you may need to opt-into that (it will keep backward compatibility with old correlation protocol supported by ApplicationInsights SDKs).
```

Comment:
in latest version - there is no need to opt-in, right?

Reply:
there are 3 languages covered in this doc, some of them require to opt-in